### DISCUSSION TITLE
Withdraw from GitHub Octernships Projec

### DISCUSSION THREAD

**Discussion by sshdopey** [community/community#49524] [GitHub Education]
If I mistakenly apply (click "I'm interested") is there a way to revert that decision and show interest in a new octernship option? I have already used my four available options.

**Comment by divais** [comment]
> If I mistakenly apply (click "I'm interested") is there a way to revert that decision and show interest in a new octernship option? I have already used my four available options.

Thank you for the ask, we dont have this capability currently but will add it to our backlog. 
Please upvote to help us understand how many students need it.

**Reply by Judge-Paul** [reply]
Hey I was wondering if it's an option now cause I made the same mistake and the company uses a stack I don't

**Comment by sshdopey** [comment]
Thank you very much. another question. If I complete all the project perfectly, would i get the octernship with the four partners?

**Reply by ashwinexe** [reply]
It'd be awesome if you get offers from all four Octernships you applied, it is possible. But you can **only accept one**.
